CONTRIBUTING.md

```diff
@@ -317,6 +317,8 @@ GitHub provides reactions as a simple and quick way to provide feedback to pull
 
 Pull request reviews should include a list of what has been reviewed in a comment, so other reviewers and mergers can know the state of the review.
 
+Pull request reviews that only nitpick things that are easily fixed in treewides cause unnecessary friction and are better left unposted. Nitpicks may however accompany blockers.
```

Comment:
Some random ideas, mix'n'matched

```suggestion
Pull request reviews that only ask for non-functional ("cosmetic", "stylistic", "meta") changes ("nitpicks") are better left unposted to reduce friction and avoid stalling the PR. Decisions about stylistic changes are better made in global, "treewide", commits and enforced using CI. A "nitpick" is never a blocker, but may accompany a blocker.
```

Reply:
```suggestion
Be considerate about posting reviews that only ask for changes are stylistic in nature or easily fixed in treewides, so as to avoid stalling the PR. Nitpicks may however accompany blockers.
```